src/Classes/ServiceAPI/MyRadio_Webcam.php

```diff
@@ -69,32 +69,39 @@ public static function getArchiveTimeRange()
      */
     public static function getCurrentWebcam()
     {
-        $current = file_get_contents(Config::$webcam_current_url);
+        if (Config::$webcam_current_url) {
+            $current = (int)file_get_contents(Config::$webcam_current_url);
```

Comment:
Will this cause an exception/error if the file actually contains a string (ie a URL) as that is what the default fall-through is there for - yes it's a horrible hack, and the form to allow setting a URL is not available to users.